piper_description/urdf/piper.ros2_control.xacro

```diff
@@ -8,6 +8,7 @@
     mock_hardware:=false
     fake_sensor_commands:=false
     sim_gazebo:=false
+    sim_ignition:=false

```

Comment:
`ignition` is a deprecated term. I use `sim_gazebo` here.

Reply:
it is used in the macro down below, I added it for it to not crash

Reply:
Yep it needs to be removed from line 32.

Reply:
```suggestion

```